app/controllers/index.js

```diff
@@ -10,7 +10,6 @@ const BASE_URL = ENV.BASE_API_URL;
 
 export default class IndexController extends Controller {
   @service toast;
-  @tracked status = this.model;
```

Comment:
Why we have removed this, I think it's used below ?

Reply:
check mobile.js for more context

Reply:
I think this is different and was not added by you in `Index Controller`, this is being used for tracking the status of the user whether they are ooo, active , or idle

Reply:
agreed